PurchasesCoreSwift/Logging/Logger.swift

```diff
@@ -29,20 +29,59 @@ import Foundation
         guard level != .debug || shouldShowDebugLogs else { return }
         NSLog("[\(frameworkDescription)] - \(level.description()): \(message)")
     }
+    
+    static func log(level: LogLevel, intent: LogIntent, message: String) {
+        #if V2_LOGS_ENABLED
+            let messageWithPrefix = "\(intent.suffix) \(message)"
+            Logger.log(level: level, message: messageWithPrefix)
+        #else
+            Logger.log(level: level, message: message)
+        #endif
+    }
 
     static func debug(_ message: String) {
-        log(level: .debug, message: message)
+        log(level: .debug, intent: .info, message: message)
     }
 
     static func info(_ message: String) {
-        log(level: .debug, message: message)
+        log(level: .info, intent: .info, message: message)
     }
 
     static func warn(_ message: String) {
-        log(level: .debug, message: message)
+        log(level: .warn, intent: .warning, message: message)
     }
 
     static func error(_ message: String) {
-        log(level: .debug, message: message)
+        log(level: .error, message: message)
+    }
+}
+
+extension Logger {
```

Comment:
this probably needs to be `@objc public` in order for obj-c to see it

Reply:
Adding the `@objc public` decorator could be applied to the extension or to the methods in the extension, either or would work. I will go with the former.